shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/prepare/datasource/PrepareTargetSchemasParameter.java

```diff
@@ -30,7 +30,7 @@
 @Getter
 public final class PrepareTargetSchemasParameter {
     
-    private final TaskConfiguration taskConfig;
+    private final RuleAlteredJobConfiguration jobConfig;
     
```

Comment:
It's better not depend on RuleAlteredJobConfiguration, since it's dedicated for a type of job. PrepareTargetSchemasParameter is planned for common usage (it might be used by several types of jobs).

For example:
- Add logicTableNames in PrepareTargetSchemasParameter to replace parameter.getJobConfig().splitLogicTableNames()
- Extract sourceDatabaseType.isSchemaAvailable() to RuleAlteredJobPreparer

And also PrepareTargetTablesParameter.

Reply:
get it